### DISCUSSION TITLE
Accidentally Bricked My YI Home (IFUS) Camera - Please help.

### DISCUSSION THREAD

**Discussion by sphi-nx** [roleoroleo/yi-hack-Allwinner-v2#290] [Q&A]
Really grateful for @roleoroleo for this hack.

Sorry, noob mistake here. I have five Yi Home cameras purchased a few months apart from Amazon. Two are IFUS (firmware 9.0.36*) and the other three are BFUS. I had already successfully flashed the BFUS and was using the yi-hack-Allwinner-v2 to flash the IFUS camera but accidentally copied the y21ga instead of the y211ga to the formatted SD Card. Now my IFUS camera is bricked with steady orange light.

I want to follow [this guide](https://github.com/roleoroleo/yi-hack-Allwinner-v2/wiki/Unbrick-the-cam) to try to unbrick it but need some pointers;

1. Is [this](https://www.konga.com/product/arduino-pl2303-usb-to-rs232-ttl-stc-microcontroller-programmer-4819800) the adapter I need to get?
2. Is there any other guide or video that anyone can share to fix this?

Any help will be greatly appreciated. 

Thank you.

**Comment by roleoroleo** [comment]
Before to start with the serial interface, please try to use this unbrick archive for your y211ga:
[unbrick_y211ga.tar.gz](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/7409639/unbrick_y211ga.tar.gz)
Use it like a normal hack.

**Reply by sphi-nx** [reply]
Hi @roleoroleo . Thanks so much for the reply.

The weirdest thing happened. When I got back home and plugged the camera to power, it worked. I've powered it off and on again and it's working well. It seems to have fixed itself.

I'm happy but curious if this common.
